In [15]:
import numpy as np
import jieba
import pandas as pd
from gensim import corpora,models
import gensim



In [16]:

inputfile = "resultneat.csv"
outputfile = "resultcyt.csv"
data = pd.read_csv(inputfile,encoding='utf8')
def mycut(s):return " ".join(jieba.cut(s))
data1 = data['标题／微博内容'].astype(str).apply(mycut)
data1.to_csv(outputfile,index=False,header=False)
    

In [17]:

anatext = pd.read_csv("resultcyt.csv",header=None)
stoplist = 'stoplist.txt'
stop = pd.read_csv(stoplist,header=None,sep ='tipdm',engine='python')
stop = [" ",""] + list(stop[0])
# 定义分隔函数，然后用apply进行广播
anatext[1] = anatext[0].astype(str).apply(lambda s: s.split(" "))
anatext[2] = anatext[1].apply(lambda x: [i for i in x if i not in stop])
#  怎么建立语料库？？？
 # 建立词典
stop_dict = corpora.Dictionary(anatext[2])
# 建立语料库
stop_corpus = [stop_dict.doc2bow(i) for i in anatext[2]]


In [18]:
# LDA模型训练
LDA_model = models.LdaModel(stop_corpus,num_topics = 10,id2word = stop_dict)
for i in range(10):
    print(LDA_model.print_topic(i))

0.029*"求" + 0.024*"专家" + 0.023*"开窗" + 0.020*"时" + 0.019*"时间" + 0.015*"通风" + 0.014*"不语" + 0.013*"上午" + 0.012*"作揖" + 0.012*"空气污染"
0.019*"疫情" + 0.014*"空气污染" + 0.013*"空气" + 0.011*"污染" + 0.010*"日" + 0.010*"中" + 0.009*"PM2.5" + 0.008*"影响" + 0.008*"出现" + 0.007*"月"
0.036*"病毒" + 0.022*"泪" + 0.021*"吃" + 0.020*"肺炎" + 0.009*"医院" + 0.008*"中年" + 0.008*"肺" + 0.007*"医生" + 0.007*"非典" + 0.006*"铁"
0.052*"转发" + 0.048*"微博" + 0.018*"导航" + 0.016*"自动" + 0.016*"系统" + 0.015*"PM2.5" + 0.012*"早日" + 0.010*"关注" + 0.008*"禁" + 0.007*"价格"
0.016*"2020" + 0.015*"污染" + 0.014*"年" + 0.010*"月" + 0.010*"烟花爆竹" + 0.010*"爆竹" + 0.009*"工作" + 0.009*"燃放烟花" + 0.009*"大气污染" + 0.008*"nan"
0.064*"笑" + 0.033*"doge" + 0.032*"真相" + 0.032*"哈哈哈" + 0.029*"爱" + 0.028*"cry" + 0.027*"就会" + 0.019*"山火" + 0.017*"死" + 0.014*"疑问"
0.115*"霾" + 0.095*"雾" + 0.015*"武汉" + 0.014*"北京" + 0.011*"允悲" + 0.011*"今天" + 0.010*"严重" + 0.009*"天" + 0.007*"河北" + 0.007*"好"
0.016*"好" + 0.010*"一个" + 0.010*"想" + 0.008*"现在" + 0.007*"希望" + 0.007*"知道" + 0.007*"真的" + 0.007*"加油" 

In [19]:
LDA_model.save('poll_model.model')

In [20]:
new_model = models.ldamodel.LdaModel.load('poll_model.model')

In [20]:
HDP_Model = models.HdpModel(stop_coropus,id2word = stop_dict )


In [ ]:
print("hello world")

In [21]:
import pyLDAvis
import gensim
import pyLDAvis.gensim

In [22]:

pyLDAvis.enable_notebook()
vis = pyLDAvis.gensim.prepare(LDA_model,stop_corpus,stop_dict)
vis

PreparedData(topic_coordinates=              x         y  topics  cluster       Freq
topic                                                
7      0.212881 -0.202430       1        1  21.374245
6      0.191459 -0.218260       2        1  17.335119
4      0.114326  0.279434       3        1  17.281073
1      0.143496  0.194462       4        1  13.608556
9      0.162911  0.085074       5        1  13.292522
2      0.048013 -0.121503       6        1   6.700019
3     -0.142648  0.108905       7        1   3.982829
0     -0.214159  0.007007       8        1   2.383788
5     -0.252365 -0.084719       9        1   2.132977
8     -0.263914 -0.047970      10        1   1.908873, topic_info=     Term           Freq          Total Category  logprob  loglift
8       霾  214871.000000  214871.000000  Default  30.0000  30.0000
7       雾  175496.000000  175496.000000  Default  29.0000  29.0000
390     心   21964.000000   21964.000000  Default  28.0000  28.0000
610    转发   21466.000000   21466.000000  Default  27.0000  27.0000
9937   病毒   27072.000000   27072.000000  Default  26.0000  26.0000
...   ...            ...            ...      ...      ...      ...
2583   新鲜    1593.151001    1899.093262  Topic10  -4.8227   3.7830
4971   分享    1677.246338    2483.683838  Topic10  -4.7712   3.5661
1190   拍摄     817.684509    1131.913696  Topic10  -5.4897   3.6335
5682    狗     928.884705    1788.950928  Topic10  -5.3621   3.3033
1742    良     742.500366    1291.539062  Topic10  -5.5861   3.4051

[589 rows x 6 columns], token_table=       Topic      Freq    Term
term                          
12633      6  0.999269  ......
173        3  0.409803      11
173        4  0.200699      11
173        7  0.047561      11
173        8  0.341537      11
...      ...       ...     ...
4293       2  0.968660      鲜花
6913       2  0.999495      鼓掌
456        7  0.157863       鼻
456        9  0.841725       鼻
777        6  0.999208      鼻炎

[1118 rows x 3 columns], R=30, lambda_step=0.01, plot_opts={'xlab': 'PC1', 'ylab': 'PC2'}, topic_order=[8, 7, 5, 2, 10, 3, 4, 1, 6, 9])

In [21]:
for i in range(0,70):
    print(HDP_Model.print_topic(i))

# topic_result = []
# topic_result = HDP_Model.print_topics
# for i in topic_result:
#     print (i)

0.005*"月" + 0.005*"年" + 0.005*"霾" + 0.004*"污染" + 0.004*"雾" + 0.004*"工作" + 0.004*"日" + 0.004*"大气污染" + 0.003*"中" + 0.003*"防治"
0.010*"霾" + 0.008*"雾" + 0.005*"中国" + 0.005*"一个" + 0.004*"好" + 0.004*"中" + 0.004*"印度" + 0.003*"吃" + 0.003*"年" + 0.002*"里"
0.023*"霾" + 0.020*"雾" + 0.008*"好" + 0.007*"一个" + 0.006*"今天" + 0.006*"想" + 0.005*"真的" + 0.005*"吃" + 0.004*"信仰" + 0.003*"天"
0.022*"霾" + 0.016*"雾" + 0.013*"日" + 0.007*"月" + 0.007*"天气" + 0.007*"污染" + 0.006*"空气污染" + 0.005*"12" + 0.005*"空气质量" + 0.005*"空气"
0.033*"霾" + 0.028*"雾" + 0.007*"好" + 0.006*"今天" + 0.004*"天" + 0.004*"一个" + 0.004*"真的" + 0.004*"想" + 0.004*"吃" + 0.004*"北京"
0.027*"霾" + 0.023*"雾" + 0.007*"好" + 0.004*"今天" + 0.003*"真的" + 0.003*"心" + 0.003*"一个" + 0.003*"天" + 0.003*"北京" + 0.003*"想"
0.034*"霾" + 0.030*"雾" + 0.008*"好" + 0.006*"今天" + 0.005*"真的" + 0.004*"太" + 0.004*"天" + 0.004*"想" + 0.003*"一个" + 0.003*"天气"
0.021*"霾" + 0.017*"雾" + 0.016*"生态" + 0.009*"币" + 0.005*"the" + 0.005*"好" + 0.004*"to" + 0.004*"一个" + 0.003*"需要" + 0.003*"of"
0.020*"霾" + 0.

0.025*"霾" + 0.021*"雾" + 0.004*"好" + 0.004*"天" + 0.003*"微博" + 0.003*"nan" + 0.003*"转发" + 0.003*"今天" + 0.002*"天气" + 0.002*"太"
0.026*"霾" + 0.022*"雾" + 0.004*"好" + 0.004*"微博" + 0.003*"转发" + 0.003*"天" + 0.003*"nan" + 0.003*"早安" + 0.003*"刘瑞琦" + 0.003*"发展中国家"
0.027*"霾" + 0.024*"雾" + 0.009*"居家" + 0.007*"天气" + 0.005*"简单" + 0.005*"危害" + 0.005*"中医" + 0.005*"值得" + 0.005*"造成" + 0.005*"锻炼"
0.021*"霾" + 0.018*"雾" + 0.005*"指数" + 0.005*"倍" + 0.005*"胡子" + 0.005*"空气" + 0.004*"污染" + 0.004*"普通" + 0.003*"留有" + 0.003*"好"


In [8]:
for i in range(20):
    print(LDA_model.print_topic(i))

0.297*"霾" + 0.249*"雾" + 0.032*"河北" + 0.029*"天" + 0.021*"严重" + 0.015*"北京" + 0.014*"感冒" + 0.011*"天津" + 0.010*"雷达" + 0.008*"防雾"
0.109*"呼吸" + 0.072*"肺" + 0.068*"空气污染" + 0.062*"保护" + 0.024*"喝" + 0.016*"功效" + 0.016*"春暖花开" + 0.015*"水" + 0.015*"上天" + 0.013*"棒"
0.146*"请" + 0.049*"猪" + 0.036*"分享" + 0.027*"狗" + 0.026*"新鲜" + 0.022*"解答" + 0.022*"雨水" + 0.015*"密集" + 0.014*"答案" + 0.012*"难忘"
0.069*"病毒" + 0.035*"肺炎" + 0.018*"健康" + 0.017*"感染" + 0.016*"疾病" + 0.015*"医院" + 0.013*"空气污染" + 0.012*"医生" + 0.012*"预防" + 0.010*"咳嗽"
0.080*"年" + 0.041*"微风" + 0.036*"屁" + 0.034*"作揖" + 0.031*"新" + 0.025*"味道" + 0.025*"左" + 0.025*"柑" + 0.021*"2019" + 0.020*"朋友"
0.014*"环保" + 0.013*"新型" + 0.012*"限制" + 0.011*"驾驶" + 0.011*"年" + 0.011*"证明" + 0.010*"汽车" + 0.009*"安全" + 0.008*"规定" + 0.008*"农村"
0.123*"爱" + 0.036*"good" + 0.029*"带走" + 0.021*"男人" + 0.018*"越" + 0.014*"儿" + 0.014*"转基因" + 0.014*"民众" + 0.011*"挤眼" + 0.010*"麻烦"
0.038*"霾" + 0.035*"心" + 0.034*"太阳" + 0.026*"雾" + 0.024*"阳光" + 0.021*"散" + 0.020*"希望" + 0.016*"温暖" + 0.013*"回来" +

In [4]:
data = pd.read_csv('resultneat.csv',header = None)
data

0      1  \
0                                                 标题／微博内容  原创/转发   
1                                   这几天正好偏北风，吹来了雾霾天[笑cry]     转发   
2                                        希望河北省早点禁行北京车牌的车辆     转发   
3                                            可惜都是雾霾[汗][怒]     转发   
4                                   这几天正好偏北风，吹来了雾霾天[笑cry]     转发   
...                                                   ...    ...   
403185                              四川马上复工了，延迟假期的概念都搞清楚了吗     转发   
403186                                    小时候的春节 就是这样的[泪]     转发   
403187  最近路上车也少了，企业也都停产了，工地也停工了，一切娱乐活动也没有了可为啥雾霾还这么严重呢能...     原创   
403188  武汉新冠病毒也把环保专家们多年来所谓雾霾形成原因与机制重大研究成果击的粉碎初二起北京持续雾霾...     原创   
403189                                          那个时候没有雾霾。     转发   

                          2  \
0                      发布日期   
1       2019-11-01 00:00:00   
2       2019-11-01 00:00:00   
3       2019-11-01 00:00:00   
4       2019-11-01 00:02:00   
...                     ...   
403185  2020-01-30 23:56:00   
403186  2020-01-30 23:56:00   
403187  2020-01-30 23:57:00   
403188  2020-01-30 23:57:00   
403189  2020-01-30 23:57:00   

                                                        3     4   5   6     7  
0                                                   原微博内容  认证类型  地域  性别  微博情绪  
1       @上海生活头条资讯 #上海身边事# 【乘坐直升机从高空俯瞰上海是什么样子的？】第二届进博会开...    普通  山东   男    喜悦  
2       @南方周末 【被“进京证”困住的北京人：一年只准84天，通勤怎么办？】2019年11月1日起...    普通  北京   男    愤怒  
3       @上海生活头条资讯 #上海身边事# 【乘坐直升机从高空俯瞰上海是什么样子的？】第二届进博会开...    普通  浙江   男    喜悦  
4       @上海生活头条资讯 #上海身边事# 【乘坐直升机从高空俯瞰上海是什么样子的？】第二届进博会开...    普通  吉林   男    喜悦  
...                                                   ...   ...  ..  ..   ...  
403185  @人力资源研究 律师关于本次延长休假的几条法律解释[雾霾]图1?:延长休假是强制的吗？企业遵...    普通  四川   男    喜悦  
403186  @最神奇的视频 网友：这是03年拍的视频，那年我18岁，现在的我把这视频给我女儿看，告诉她，...    普通  上海   男    喜悦  
403187                                                       普通  浙江   女    喜悦  
403188                                                       普通  山东   男    中性  
403189  @最神奇的视频 网友：这是03年拍的视频，那年我18岁，现在的我把这视频给我女儿看，告诉她，...    达人  北京   男    喜悦  

[403190 rows x 8 columns]

* 分析结果


0.015*"春节" + 0.014*"转发" + 0.013*"烟花爆竹" + 0.013*"微博" + 0.012*"爆竹" + 0.012*"污染" + 0.011*"工作" + 0.011*"燃放烟花" + 0.011*"大气污染" + 0.010*"过年"
0.029*"空气污染" + 0.018*"肺" + 0.017*"健康" + 0.016*"空气" + 0.016*"大脑" + 0.016*"时" + 0.016*"疾病" + 0.015*"中年" + 0.015*"就会" + 0.014*"真相"
0.163*"霾" + 0.134*"雾" + 0.069*"“" + 0.068*"”" + 0.020*"天" + 0.013*"微笑" + 0.011*"严重" + 0.011*"北京" + 0.010*"感冒" + 0.009*"雷达"
0.038*"霾" + 0.031*"雾" + 0.016*"t" + 0.014*"http" + 0.013*"好" + 0.007*"武汉" + 0.007*"想" + 0.007*"一个" + 0.007*"·" + 0.007*"现在"
0.024*"泪" + 0.017*"加油" + 0.011*"地球" + 0.010*"空气" + 0.009*"净化器" + 0.007*"流" + 0.006*"眼泪" + 0.006*"防雾" + 0.006*"石家庄" + 0.006*"隔离"
0.028*"中国" + 0.015*"人类" + 0.013*"问题" + 0.012*"一个" + 0.012*"疫情" + 0.012*"赞" + 0.008*"需要" + 0.008*"国家" + 0.008*"一种" + 0.008*"社会"
0.061*"笑" + 0.031*"求" + 0.030*"哈哈哈" + 0.027*"cry" + 0.024*"新年" + 0.019*"鲜花" + 0.015*"微风" + 0.014*"不语" + 0.014*"关注" + 0.012*"作揖"
0.017*"年" + 0.014*"污染" + 0.012*"月" + 0.011*"日" + 0.010*"PM2.5" + 0.010*"2020" + 0.009*"空气污染" + 0.006*"空气质量" + 0.006*"造成" + 0.006*"影响"
0.044*"平安" + 0.031*"回复" + 0.019*"証" + 0.017*"稀少" + 0.016*"成语" + 0.012*"新鲜" + 0.012*"解答" + 0.012*"雨水" + 0.010*"掦" + 0.010*"鬼"
0.045*"病毒" + 0.022*"心" + 0.020*"肺炎" + 0.019*"吃" + 0.018*"nan" + 0.013*"冠状病毒" + 0.011*"感染" + 0.008*"喵" + 0.008*"嘻嘻" + 0.006*"医院"


0.061*"PM2.5" + 0.033*"空气" + 0.022*"净化器" + 0.019*"pm2.5" + 0.016*"雷达" + 0.015*"鲜花" + 0.014*"空气质量" + 0.013*"中" + 0.013*"good" + 0.012*"地图"
0.018*"疫情" + 0.015*"”" + 0.015*"“" + 0.009*"问题" + 0.008*"人类" + 0.008*"中国" + 0.007*"国家" + 0.007*"中" + 0.007*"证明" + 0.006*"环保"
0.028*"空气污染" + 0.025*"直升机" + 0.017*"t" + 0.013*"http" + 0.013*"中国" + 0.013*"空气" + 0.013*"严重" + 0.012*"设" + 0.010*"指数" + 0.009*"级"
0.017*"污染" + 0.015*"2020" + 0.015*"月" + 0.014*"日" + 0.014*"年" + 0.010*"烟花爆竹" + 0.009*"爆竹" + 0.009*"燃放烟花" + 0.008*"大气污染" + 0.008*"工作"
0.048*"春节" + 0.044*"转发" + 0.040*"微博" + 0.031*"赞" + 0.024*"新型" + 0.021*"中国" + 0.014*"关注" + 0.014*"3D" + 0.011*"自动" + 0.008*"支持"
0.097*"霾" + 0.080*"雾" + 0.025*"t" + 0.022*"http" + 0.014*"好" + 0.012*"武汉" + 0.011*"今天" + 0.011*"·" + 0.010*"北京" + 0.009*"允悲"
0.131*"心" + 0.044*"就会" + 0.023*"猪" + 0.019*"小心" + 0.014*"成语" + 0.012*"分享" + 0.011*"想着" + 0.010*"雨水" + 0.009*"瞎" + 0.008*"春色"
0.013*"一个" + 0.011*"现在" + 0.008*"好" + 0.007*"做" + 0.007*"想" + 0.007*"知道" + 0.007*"里" + 0.006*"吃" + 0.006*"看到" + 0.006*"再"
0.140*"“" + 0.140*"”" + 0.057*"nan" + 0.030*"爱" + 0.025*"平安" + 0.015*"鼻" + 0.014*"点" + 0.014*"作揖" + 0.009*"干" + 0.007*"米奇"
0.026*"病毒" + 0.013*"肺炎" + 0.008*"时" + 0.008*"空气污染" + 0.007*"健康" + 0.007*"冠状病毒" + 0.007*"感染" + 0.006*"肺" + 0.006*"空气" + 0.005*"吃"

.028*"肺炎" + 0.026*"吃" + 0.014*"doge" + 0.008*"流" + 0.007*"爱" + 0.007*"早日" + 0.006*"微风" + 0.006*"身体" + 0.006*"喝" + 0.006*"流感"
0.080*"病毒" + 0.025*"冠状病毒" + 0.020*"感染" + 0.020*"肺" + 0.018*"非典" + 0.016*"疾病" + 0.013*"钟南山" + 0.012*"咳嗽" + 0.011*"防雾" + 0.011*"预防"
0.018*"污染" + 0.016*"春节" + 0.014*"烟花爆竹" + 0.013*"爆竹" + 0.012*"燃放烟花" + 0.011*"大气污染" + 0.011*"工作" + 0.009*"安全" + 0.009*"月" + 0.008*"禁止"
0.150*"霾" + 0.131*"雾" + 0.016*"北京" + 0.013*"今天" + 0.012*"天" + 0.011*"严重" + 0.010*"好" + 0.010*"太阳" + 0.007*"终于" + 0.007*"太"
0.125*"心" + 0.025*"回复" + 0.017*"肆虐" + 0.014*"成语" + 0.011*"分享" + 0.010*"绕" + 0.010*"雨水" + 0.010*"解答" + 0.009*"新鲜" + 0.009*"网友"
0.052*"出门" + 0.051*"赞" + 0.022*"鲜花" + 0.014*"作揖" + 0.013*"憧憬" + 0.011*"PM2.5" + 0.011*"汗" + 0.010*"2020.1" + 0.010*"防" + 0.010*"左"
0.020*"武汉" + 0.017*"疫情" + 0.017*"中国" + 0.013*"空气污染" + 0.009*"中" + 0.009*"人类" + 0.009*"空气" + 0.008*"问题" + 0.008*"造成" + 0.007*"污染"
0.013*"好" + 0.012*"一个" + 0.009*"现在" + 0.007*"想" + 0.007*"真的" + 0.006*"知道" + 0.006*"加油" + 0.006*"做" + 0.006*"里" + 0.005*"希望"
0.030*"年" + 0.016*"2020" + 0.011*"nan" + 0.010*"过年" + 0.008*"新型" + 0.007*"PM2.5" + 0.007*"全国" + 0.006*"城市" + 0.006*"系统" + 0.005*"新年"
0.021*"霾" + 0.019*"允悲" + 0.018*"西安" + 0.018*"泪" + 0.017*"河北" + 0.016*"日" + 0.014*"天气" + 0.013*"月" + 0.011*"空气" + 0.009*"重度"


0.021*"日" + 0.021*"月" + 0.019*"年" + 0.017*"2020" + 0.014*"nan" + 0.013*"PM2.5" + 0.012*"污染" + 0.012*"空气质量" + 0.010*"空气污染" + 0.009*"天气"
0.015*"病毒" + 0.013*"疫情" + 0.008*"空气" + 0.008*"中" + 0.008*"空气污染" + 0.007*"人类" + 0.007*"问题" + 0.006*"造成" + 0.005*"需要" + 0.005*"健康"
0.056*"转发" + 0.052*"微博" + 0.028*"肺" + 0.022*"肺炎" + 0.018*"咳嗽" + 0.018*"吃" + 0.014*"good" + 0.009*"肆虐" + 0.009*"茶" + 0.008*"样子"
0.031*"中国" + 0.027*"笑" + 0.013*"确认" + 0.012*"求" + 0.012*"cry" + 0.012*"烟花" + 0.010*"平安" + 0.009*"关注" + 0.007*"这段话" + 0.006*"不语"
0.107*"加油" + 0.068*"赞" + 0.049*"感染" + 0.044*"爱" + 0.029*"鲜花" + 0.021*"微风" + 0.020*"拜拜" + 0.017*"憧憬" + 0.016*"鼓掌" + 0.012*"色"
0.036*"春节" + 0.031*"烟花爆竹" + 0.029*"爆竹" + 0.026*"燃放烟花" + 0.018*"安全" + 0.017*"燃放" + 0.015*"驾驶" + 0.014*"鞭炮" + 0.013*"时" + 0.011*"禁止"
0.016*"污染" + 0.011*"工作" + 0.010*"大气污染" + 0.010*"年" + 0.008*"月" + 0.007*"防治" + 0.006*"2020" + 0.006*"环保" + 0.006*"机动车" + 0.006*"限制"
0.013*"一个" + 0.013*"好" + 0.009*"现在" + 0.008*"想" + 0.008*"吃" + 0.007*"知道" + 0.007*"里" + 0.006*"做" + 0.006*"再" + 0.006*"看到"
0.103*"心" + 0.034*"pm2.5" + 0.021*"回复" + 0.015*"小心" + 0.013*"照片" + 0.012*"网友" + 0.012*"点" + 0.011*"証" + 0.011*"成语" + 0.010*"躲避"
0.130*"霾" + 0.106*"雾" + 0.016*"武汉" + 0.013*"北京" + 0.013*"允悲" + 0.011*"好" + 0.011*"今天" + 0.010*"严重" + 0.010*"天" + 0.009*"西安"


* 20个主题的分析结果  
0.050*"泪" + 0.050*"空气" + 0.038*"PM2.5" + 0.020*"净化器" + 0.018*"pm2.5" + 0.012*"需要" + 0.012*"隔离" + 0.010*"病毒" + 0.009*"效果" + 0.009*"净化"
0.095*"年" + 0.038*"2020" + 0.018*"新型" + 0.016*"市场" + 0.016*"2019" + 0.013*"中国" + 0.013*"大会" + 0.013*"关注" + 0.012*"汽车" + 0.012*"国家"
0.025*"中国" + 0.024*"一个" + 0.016*"人类" + 0.013*"现在" + 0.012*"做" + 0.012*"世界" + 0.009*"一种" + 0.009*"思考" + 0.008*"地球" + 0.008*"变成"
0.013*"中" + 0.011*"城市" + 0.009*"驾驶" + 0.009*"发展" + 0.009*"问题" + 0.007*"地形" + 0.007*"后" + 0.007*"环保" + 0.007*"安全" + 0.007*"系统"
0.127*"请" + 0.059*"动物" + 0.035*"猪" + 0.031*"分享" + 0.030*"样子" + 0.024*"躲避" + 0.023*"狗" + 0.020*"想着" + 0.016*"雨水" + 0.015*"欺"
0.119*"武汉" + 0.043*"太阳" + 0.040*"感冒" + 0.034*"大脑" + 0.022*"不敢" + 0.021*"注意" + 0.020*"度" + 0.019*"good" + 0.018*"传播" + 0.014*"空气"
0.067*"西安" + 0.028*"屋里" + 0.028*"真相" + 0.019*"生命" + 0.018*"人生" + 0.016*"希望" + 0.013*"成功" + 0.012*"值得" + 0.012*"知道" + 0.012*"生活"
0.045*"病毒" + 0.028*"空气污染" + 0.017*"污染" + 0.015*"健康" + 0.014*"造成" + 0.012*"中" + 0.012*"疾病" + 0.011*"证明" + 0.011*"专家" + 0.011*"原因"
0.048*"心" + 0.043*"吃" + 0.022*"铁" + 0.021*"睡" + 0.020*"太" + 0.020*"求" + 0.015*"开心" + 0.013*"一起" + 0.012*"晚上" + 0.010*"防护"
0.079*"感染" + 0.074*"肺" + 0.043*"保护" + 0.036*"呼吸道" + 0.017*"传染病" + 0.017*"四天" + 0.016*"鼻炎" + 0.015*"米奇" + 0.014*"蜡烛" + 0.013*"加重"
0.032*"霾" + 0.024*"好" + 0.023*"雾" + 0.014*"想" + 0.013*"真的" + 0.010*"现在" + 0.009*"今天" + 0.009*"一个" + 0.008*"知道" + 0.008*"转发"
0.017*"年" + 0.015*"平安" + 0.013*"2020" + 0.013*"新" + 0.013*"鲜花" + 0.010*"温暖" + 0.010*"明" + 0.010*"微风" + 0.009*"可怕" + 0.009*"支持"
0.043*"月" + 0.041*"日" + 0.025*"空气质量" + 0.023*"过年" + 0.023*"赞" + 0.019*"空气污染" + 0.018*"年" + 0.018*"时" + 0.014*"污染" + 0.013*"指数"
0.242*"北京" + 0.046*"死" + 0.044*"回来" + 0.039*"禁放" + 0.038*"上海" + 0.023*"作揖" + 0.020*"广州" + 0.011*"煤" + 0.011*"印度" + 0.010*"数值"
0.033*"污染" + 0.032*"工作" + 0.024*"大气污染" + 0.021*"环保" + 0.020*"防治" + 0.017*"治理" + 0.015*"防控" + 0.012*"情况" + 0.011*"生态环境" + 0.010*"工地"
0.307*"霾" + 0.261*"雾" + 0.029*"天气" + 0.025*"河北" + 0.024*"严重" + 0.023*"天" + 0.018*"今天" + 0.011*"重度" + 0.011*"散" + 0.009*"天津"
0.170*"允悲" + 0.059*"微笑" + 0.032*"不让" + 0.025*"听见" + 0.025*"大地" + 0.015*"真是" + 0.013*"为啥" + 0.013*"色" + 0.013*"邯郸" + 0.013*"杭州"
0.027*"烟花爆竹" + 0.025*"爆竹" + 0.023*"燃放烟花" + 0.016*"污染" + 0.015*"禁止" + 0.014*"燃放" + 0.013*"机动车" + 0.013*"限制" + 0.013*"安全" + 0.011*"月"
0.053*"加油" + 0.044*"肺炎" + 0.042*"笑" + 0.018*"cry" + 0.013*"一会" + 0.013*"流" + 0.011*"大自然" + 0.010*"天空" + 0.010*"早日" + 0.010*"阳光"
0.082*"爱" + 0.036*"点" + 0.036*"大雾" + 0.020*"干" + 0.020*"机场" + 0.018*"美" + 0.018*"字" + 0.017*"网友" + 0.014*"兔子" + 0.013*"国际"


0.133*"心" + 0.029*"good" + 0.023*"猪" + 0.019*"预报" + 0.015*"成语" + 0.012*"新鲜" + 0.012*"分享" + 0.011*"稀释" + 0.011*"AQI" + 0.010*"解答"
0.018*"疫情" + 0.013*"中" + 0.011*"空气污染" + 0.009*"人类" + 0.008*"病毒" + 0.007*"需要" + 0.007*"造成" + 0.006*"问题" + 0.006*"出现" + 0.006*"中国"
0.014*"好" + 0.012*"一个" + 0.009*"现在" + 0.007*"想" + 0.007*"吃" + 0.007*"真的" + 0.007*"知道" + 0.006*"里" + 0.006*"希望" + 0.005*"再"
0.018*"空气" + 0.013*"PM2.5" + 0.012*"喵" + 0.009*"澳大利亚" + 0.009*"石家庄" + 0.008*"早日" + 0.007*"澳洲" + 0.007*"怒" + 0.006*"疑问" + 0.006*"消毒"
0.012*"春节" + 0.011*"污染" + 0.011*"转发" + 0.010*"烟花爆竹" + 0.009*"爆竹" + 0.009*"工作" + 0.009*"燃放烟花" + 0.008*"环保" + 0.008*"大气污染" + 0.007*"微博"
0.179*"霾" + 0.149*"雾" + 0.019*"北京" + 0.016*"允悲" + 0.015*"严重" + 0.014*"天" + 0.014*"西安" + 0.013*"泪" + 0.012*"河北" + 0.011*"今天"
0.027*"月" + 0.025*"日" + 0.024*"年" + 0.021*"2020" + 0.019*"污染" + 0.014*"PM2.5" + 0.014*"天气" + 0.013*"空气质量" + 0.010*"新型" + 0.008*"空气污染"
0.041*"病毒" + 0.034*"肺炎" + 0.026*"nan" + 0.024*"赞" + 0.014*"冠状病毒" + 0.014*"中年" + 0.012*"开窗" + 0.012*"平安" + 0.012*"时" + 0.011*"净化器"
0.035*"武汉" + 0.017*"笑" + 0.013*"太阳" + 0.013*"过年" + 0.011*"阳光" + 0.011*"霾" + 0.009*"doge" + 0.008*"温暖" + 0.007*"cry" + 0.006*"雾"
0.097*"加油" + 0.044*"感染" + 0.026*"鲜花" + 0.025*"眼泪" + 0.020*"回复" + 0.018*"屁" + 0.017*"作揖" + 0.012*"餐饮" + 0.010*"兔子" + 0.010*"左"